<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SternaX</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionArtikal"/>
    <addaction name="actionKomintent"/>
    <addaction name="actionPriemnica"/>
    <addaction name="actionFaktura"/>
    <addaction name="actionIspretnicia"/>
    <addaction name="actionSmetka"/>
    <addaction name="actionProFaktura"/>
    <addaction name="actionPovratnica"/>
    <addaction name="actionNaracka"/>
    <addaction name="actionPrint_Form"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Izlezni Dokumenti</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionPriemnica"/>
   <addaction name="actionPovratnica"/>
   <addaction name="actionNaracka"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>Vlezni Dokumenti</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionIspretnicia"/>
   <addaction name="actionIntIspratnici"/>
   <addaction name="actionIntPriemnici"/>
   <addaction name="actionSmetka"/>
   <addaction name="actionAnalitikaArtikli"/>
   <addaction name="actionAnalitikaKomintenti"/>
   <addaction name="actionLagerLista"/>
   <addaction name="actionNalog"/>
   <addaction name="actionProFaktura"/>
   <addaction name="actionFaktura"/>
   <addaction name="actionPrint_Form"/>
  </widget>
  <action name="actionArtikal">
   <property name="text">
    <string>Artikal</string>
   </property>
  </action>
  <action name="actionKomintent">
   <property name="text">
    <string>Komintent</string>
   </property>
  </action>
  <action name="actionPriemnica">
   <property name="text">
    <string>Priemnica</string>
   </property>
  </action>
  <action name="actionFaktura">
   <property name="text">
    <string>Faktura</string>
   </property>
  </action>
  <action name="actionIspretnicia">
   <property name="text">
    <string>Ispratnicia</string>
   </property>
  </action>
  <action name="actionSmetka">
   <property name="text">
    <string>Smetka</string>
   </property>
  </action>
  <action name="actionProFaktura">
   <property name="text">
    <string>ProFaktura</string>
   </property>
  </action>
  <action name="actionPovratnica">
   <property name="text">
    <string>Povratnica</string>
   </property>
  </action>
  <action name="actionNaracka">
   <property name="text">
    <string>Naracka</string>
   </property>
  </action>
  <action name="actionPrint_Form">
   <property name="text">
    <string>Print Form</string>
   </property>
  </action>
  <action name="actionLagerLista">
   <property name="text">
    <string>LagerLista</string>
   </property>
  </action>
  <action name="actionNalog">
   <property name="text">
    <string>Nalog</string>
   </property>
  </action>
  <action name="actionIntIspratnici">
   <property name="text">
    <string>IntIspratnici</string>
   </property>
  </action>
  <action name="actionIntPriemnici">
   <property name="text">
    <string>IntPriemnici</string>
   </property>
  </action>
  <action name="actionAnalitikaKomintenti">
   <property name="text">
    <string>AnalitikaKomintenti</string>
   </property>
  </action>
  <action name="actionAnalitikaArtikli">
   <property name="text">
    <string>AnalitikaArtikli</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
